tg/update_handlers.py

```diff
@@ -43,9 +43,8 @@ def update_msg_content(controller: Controller, update: Dict[str, Any]):
     controller.model.msgs.update_msg_content(
         chat_id, message_id, update["new_content"]
     )
-    current_chat_id = controller.model.chats.id_by_index(
-        controller.model.current_chat
-    )
+
+    current_chat_id = controller.model.current_chat_id
```

Comment:
sorry it is not related to this PR but nice to improve this small bit :slightly_smiling_face: